aspnet/fundamentals/app-state/sample/src/AppState/Model/RequestEntry.cs

```diff
@@ -1,11 +1,7 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
 
 namespace AppState.Model
 {
-    [Serializable]
```

Comment:
@robbert229 Yes, since the alternative is broken everywhere. The article notes that session depends on IDistributedCache, which (except when using the memory provider, which should only be for testing) requires all data to be serialized before it can be stored. Not having serializable types in the samples would break when using e.g. a Redis or SQL Server distributed cache implementation.